README.md

```diff
@@ -38,11 +37,20 @@ This proposal presents 2 main additions to the language:
 
 ## Prior work on immutable data structures in JavaScript
 
-Today, a few libraries implement similar concepts such as [Immutable.js](https://immutable-js.github.io/immutable-js/) or [Immer](https://github.com/mweststrate/immer). Also [a previous proposal attempt](https://github.com/sebmarkbage/ecmascript-immutable-data-structures) has been done previously but abandoned because of the complexity of the proposal and lack of sufficient use cases.
+Today, a few libraries implement similar concepts such as [Immutable.js](https://immutable-js.github.io/immutable-js/). Also [a previous proposal attempt](https://github.com/sebmarkbage/ecmascript-immutable-data-structures) has been done previously but abandoned because of the complexity of the proposal and lack of sufficient use cases.
```

Comment:
```suggestion
Today, userland libraries implement similar concepts, such as [Immutable.js](https://immutable-js.github.io/immutable-js/). Also [a previous proposal attempt](https://github.com/sebmarkbage/ecmascript-immutable-data-structures) has been done previously but abandoned because of the complexity of the proposal and lack of sufficient use cases.
```

Reply:
You may want to mention immer as well - in particular I’d be interested in seeing reasons why immer isn’t sufficient.

Reply:
This is covered at the end. Immer is orthogonal

Reply:
Well, orthogonal is a bit strong, but more that immer operates on regular objects / frozen objects, and provides a pattern for pretending to mutate them through proxies and reducers, and returns frozen objects. We mention the pattern lower in the doc as a good solution that also sits well on top of records/tuples.

Reply:
yes sorry, orthogonal is indeed a big of a stretch. the point is, you could make immer output records instead of frozen objects if you wanted to